### DISCUSSION TITLE
how can i turn on this?

### DISCUSSION THREAD

**Discussion by taurynamonsterxdd** [RakambdaOrg/ChannelPointsMiner#393] [Q&A] [answered]
how can i turn on this?
![image](https://user-images.githubusercontent.com/108152893/210379143-6d4112d4-c520-4764-90fa-bea1e7c42299.png)

**Comment by taurynamonsterxdd** [comment]
ok i have https://www.youtube.com/watch?v=Fj8dKR9mzEg

**Reply by taurynamonsterxdd** [reply]
where i need put my password

**Comment by Rakambda** [comment, marked as answer]
I guess this has been answered on discord.
TV login method doesn't use any password.
